scss/components/_activity-card.scss

```diff
@@ -286,6 +286,10 @@
   background-image: url('../resources/icons/activities.svg');
 }
 
+.civicase__activity-no-result-icon--case {
```

Comment:
This is not related to activity so it should not be `.civicase__activity-no-result-icon--case`. In `_contact-cases.scss` https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/scss/components/_contact-cases-tab.scss#L43 
we already use `url('../resources/icons/cases.svg');`. 

Can we have a more generic class which can be reused at both the places?

Reply:
was agreed that the class name is correct given the circumstance, and to delete the the other class